app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :posts, class_name: 'Post', foreign_key: 'user_id'
```

Comment:
Since it's `has_many :posts`, there is no need to define the `class_name` because it will already be `Post` by default. You don't need to define the `foreign_key` either, you can [check here](https://stackoverflow.com/questions/904421/defining-foreign-key-relationships-for-rails-models) for more clarification.